src/js/components/Timestamp.js

```diff
@@ -90,6 +93,7 @@ Timestamp.propTypes = {
     PropTypes.arrayOf(FIELD_TYPES),
     FIELD_TYPES
   ]),
+  seconds: PropTypes.bool,
```

Comment:
With the field types array to achieve the full output the component would look like this 
`<Timestamp value="2016-09-27T20:01:14.468Z" fields={["date", "time", "seconds"]} />` correct? I see where you're going with this, the user should be able to isolate pieces of the Timestamp. I'll go ahead and update my PR if this sounds right to you.